pos-ui-extension-cart-line-item-details/package.json.liquid

```diff
@@ -5,14 +5,9 @@
   "version": "1.0.0",
   "license": "UNLICENSED",
   "dependencies": {
-    "react": "^18.0.0",
-    "@shopify/ui-extensions": "2025.7.x",
-    "@shopify/ui-extensions-react": "2025.7.x",
-    "react-reconciler": "0.29.0"
-  }{% if flavor contains "typescript" %},
-  "devDependencies": {
-    "@types/react": "^18.0.0"
-  }{% endif %}
+    "preact": "^10.26.9",
+    "@shopify/ui-extensions": "2025.7.x"
```

Comment:
Aren't we targeting 2025-10 for remote-dom and thus preact? In other words, we expect developers targeting 2025-07 to still be able to select and generate React extensions, and this effort looks like it would break that experience.

Just trying to get oriented to the space, but wouldn't we need to support both preact and react flavors for now?